tutorial/dotnet/realm-tutorial-dotnet/TaskPage.xaml.cs

```diff
@@ -28,12 +29,12 @@ public TaskPage()
 
         protected override async void OnAppearing()
         {
+            projectPartition = $"project={App.RealmApp.CurrentUser.Id }";
```

Comment:
Is that space needed?
```suggestion
            projectPartition = $"project={App.RealmApp.CurrentUser.Id}";
```